### PR TITLE
Add JPEG compression support

### PR DESCRIPTION
**Pull request by MattF-NSIDC**
## Description

- Add JPEG support to raster compression helper
- Compress background layer with JPEG

## Checklist

If an item on this list is done _or not needed_, simply check it with `[x]`.

- [x] Config lockfile updated (`inv config.export > qgreenland/config/cfg-lock.json`)
- [x] Version bumped if needed (`bumpversion (major|minor|patch|prerelease|build`)
- [x] CHANGELOG.md updated
- [x] Documentation updated if needed
- [x] New unit tests if needed

### CODE REVIEW

File: qgreenland/config/helpers/steps/compress_and_add_overviews.py [hunk 1]
```diff
@@ -31,13 +38,20 @@ def compress_and_add_overviews(
 
             https://gdal.org/drivers/raster/gtiff.html
     """
-    predictor_value = 3 if dtype_is_float else 2
+    compress_creation_options = [
+        '-co', 'TILED=yes',
```
Comment (trey-stafford) [inline qgreenland/config/helpers/steps/compress_and_add_overviews.py:42]: Nitpick, make this `TILED=YES`? This is how the `gdal_translate` docs suggest setting the option

Comment (MattF-NSIDC) [reply qgreenland/config/helpers/steps/compress_and_add_overviews.py]: a618192

File: qgreenland/config/helpers/steps/compress_and_add_overviews.py [hunk 2]
```diff
@@ -15,14 +15,21 @@
...
+    'JPEG',
+]
 
 
 def compress_and_add_overviews(
     *,
     input_file: str,
     output_file: str,
+    # TODO: This parameter is ignored if the compression type isn't deflate.
+    # Refactor args?
```
Comment (trey-stafford) [inline qgreenland/config/helpers/steps/compress_and_add_overviews.py:29]: We should make `dtype_is_float` `Optional`. Raise an error if it's given with a `compression_type` other than `DEFLATE`. Raise an error if it's not given with a `compression_type` of `DEFLATE`.

File: CHANGELOG.md [hunk 3]
```diff
@@ -1,6 +1,6 @@
-# NEXT_VERSION
+# v2.0.0alpha4 (Unreleased)
 
-- Update clip step in lonlat.py to solve clipping issue on lat layers
+- Resolve issue with clipping of latitude reference layers
```
Comment (trey-stafford) [inline CHANGELOG.md:3]: Let's add a note about using JPEG compression for the background layer here.

**Review by trey-stafford — APPROVED** [review]

File: CHANGELOG.md (hunk 3 above)
Comment (MattF-NSIDC) [reply CHANGELOG.md]: 63fad2a

File: qgreenland/config/helpers/steps/compress_and_add_overviews.py (hunk 2 above)
Comment (MattF-NSIDC) [reply qgreenland/config/helpers/steps/compress_and_add_overviews.py]: 032c021

**Pull request merged**